### PR TITLE
added TRUFFLE_TEST distribution

### PR DESCRIPTION
**Pull request by dougxc**

### CODE REVIEW

**Comment by rschatz** [comment]
lgtm

File: mx.truffle/mx_truffle.py [hunk 1]
```diff
@@ -93,3 +94,58 @@ def _truffle_gate_runner(args, tasks):
...
+            lines = []
+            counter = 1
+            for enum in sorted(properties.viewkeys()):
+                assert enum.startswith(metainfFile)
+                newEnum = metainfFile + str(counter)
+                counter += 1
+                for prop in properties[enum]:
+                    lines.append(newEnum + prop)
+
+            content = '\n'.join(lines)
```
Comment (chumer) [inline mx.truffle/mx_truffle.py]: Please use os.linesep instead of '\n'

Comment (rschatz) [inline mx.truffle/mx_truffle.py]: Really? That means the content of the jar file will be different depending on the platform where it was built. Isn't the jar file supposed to be platform independent?

(but probably it doesn't make any difference, the code that reads it should be robust for different line endings anyway...)

Comment (dougxc) [inline mx.truffle/mx_truffle.py]: `Properties.store0` uses the platform dependent line separator. On the other hand, when loading a properties file, Java handles both. In any case, I will implement your request Christian.

File: mx.truffle/mx_truffle.py [hunk 2]
```diff
@@ -93,3 +94,58 @@ def _truffle_gate_runner(args, tasks):
...
+    def __opened__(self, arc, srcArc, services):
+        self.settings = {}
+        self.arc = arc
+
+    def __add__(self, arcname, contents):
+        metainfFile = self._truffle_metainf_file(arcname)
+        if metainfFile:
+            properties = self.settings.setdefault(metainfFile, {})
+            for line in contents.strip().split(os.linesep):
+                if not line.startswith('#'):
```
Comment (woess) [inline mx.truffle/mx_truffle.py]: I think we shouldn't rely on the file to have windows line endings on windows.

Comment (dougxc) [inline mx.truffle/mx_truffle.py]: Good catch - will fix.

**Pull request merged**